XenAdmin/Dialogs/OptionsPages/UpdatesOptionsPage.resx

```diff
@@ -232,7 +232,7 @@
     <value>0</value>
   </data>
   <data name="AllowXenServerUpdatesCheckBox.Text" xml:space="preserve">
-    <value>Check for new versions of Xen&amp;Server</value>
+    <value>Check for &amp;new versions of @BRANDING_SERVER@</value>

```

Comment:
This duplicates the hotkey "n" (already used in AllowXenCenterUpdatesCheckBox). I suggest "v" for this one:#
Check for new &amp;versions of @BRANDING_SERVER@